azure-pipelines.yml

```diff
@@ -1,342 +1,424 @@
-              action: ${{ parameters.action }}
-              setClusterNumbersEnvVar: false
-              terraformSubscriptionID : ${{ variables.terraformSubscriptionID }}
-              ${{ if ne(parameters['cluster'], 'All') }}:
-                targetCommand: >
-                  -target azurerm_role_assignment.uami_cft_rg_identity_operator["\"${{parameters.cluster}}\""]
-                  -target azurerm_role_assignment.preview_externaldns_read_rg["\"${{parameters.cluster}}\""]
-                  -target azurerm_role_assignment.preview_externaldns_dns_zone_contributor["\"${{parameters.cluster}}\""]
-                  -target azurerm_role_assignment.preview2aat_cft_rg_identity_operator["\"${{parameters.cluster}}\""]
-                  -target data.azurerm_kubernetes_cluster.kubernetes["\"${{parameters.cluster}}\""]
-
-  - stage: BootStrapClusters
-    displayName: 'BootStrap Clusters'
-    dependsOn: Aks
-    jobs:
-      - job: BootStrap
+      - stage: "Aks_${{ component.env }}"
+        displayName: "${{ upper(component.env) }}: AKS clusters"
+        dependsOn:
+          - "CheckingClusters_${{ component.env }}"
+          - "Managed_Identity_${{ component.env }}"
+        variables:
+          existing_clusters: $[stageDependencies.CheckingClusters_${{ component.env }}.CheckingClusters.outputs['check_clusters.cluster_deploy']]
+          deploy_cluster: ${{ parameters.cluster }}
         condition: |
           or(
-            and(succeeded(), eq('${{ parameters.action }}', 'apply')),
-            and(succeeded(), eq(variables['isMain'], true), eq(variables['isAutoTriggered'], true))
+            and(eq('${{ parameters.cluster }}', 'All'), eq(variables['Build.Reason'], 'Manual')),
+            and(in(dependencies.CheckingClusters_${{ component.env }}.outputs['CheckingClusters.check_clusters.cluster_deploy'], 'All', '00', '01'), eq(variables['Build.Reason'], 'Manual')),
+            and(eq(stageDependencies.CheckingClusters_${{ component.env }}.CheckingClusters.outputs['check_clusters.cluster_deploy'], 'All'), eq(variables['Build.Reason'], 'Manual'))
           )
-        variables:
-          clusters: ${{ parameters.cluster }}
-        steps:
-          - template: pipeline-steps/bootstrap.yaml
-            parameters:
-              environment: ${{ parameters.env }}
-              cluster: ${{ parameters.cluster }}
-              project: $(project)
-              serviceConnection: $(serviceConnection)
-              aksKeyVault: $(aksKeyVault)
-              terraformSubscriptionID : $(terraformSubscriptionID)
+        jobs:
+          - job: SetClusterVariables
+            steps:
+              - script: echo "##vso[task.setvariable variable=existing_clusters;isOutput=true]value"
+                name: SetExistingClustersVariable
+              - script: echo "##vso[task.setvariable variable=deploy_cluster;isOutput=true]value"
+                name: SetDeployClusterVariable
```

Comment:
Do you need to do this? Given you already have them set in these lines above https://github.com/hmcts/aks-cft-deploy/pull/613/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R306-R308 in the variables for the stage?

Reply:
@NickAzureDevops did you mean to resolve this? I can still see them set here